main.js

```diff
@@ -84,4 +87,21 @@ document.addEventListener('DOMContentLoaded', function(e) {
             urlInputElem.value = e.target.dataset.dohurl;
         }
     });
+
+    copyRespBtn.addEventListener('click', function (e) {
+        const selectRange = document.createRange();
+        const elem = responseElem.getElementsByTagName('pre')[0];
+        if (elem) {
+            selectRange.selectNode(elem);
+            console.log('copy' + selectRange);
+            window.getSelection().removeAllRanges();
+            window.getSelection().addRange(selectRange);
+            document.execCommand("copy");
+            window.getSelection().removeAllRanges();
+        }
+        else {
+            console.log('No element to copy');
```

Comment:
This should probably be a `console.error()`, though it's not that big of a deal either way